deployer.rb

```diff
@@ -70,8 +71,14 @@ def client
     @client ||= Octokit::Client.new(:access_token => ACCESS_TOKEN)
   end
 
-  def leankit_ticket_present?(branch_name)
-    !!(branch_name =~ /(\d{6,10})/)
+  def leankit_ticket_present?(pull_request)
+    commit_messages(pull_request).any? do |comment|
+      leankit?(comment)
+    end
+  end
+
+  def leankit?(comment)
+    !!(comment =~ /(\d{6,10})/)
```

Comment:
What is this doing, exactly?

Reply:
this is poor man's identification of leankit card, just checks if there is a long number. i don't think there is any other way to identify what leankit id is